lib/block-supports/layout.php

```diff
@@ -581,11 +581,25 @@ function gutenberg_render_layout_support_flag( $block_content, $block ) {
 		$child_layout_declarations['flex-grow'] = '1';
 	}
 
-	if ( isset( $block['attrs']['style']['layout']['columnSpan'] ) ) {
+	if ( isset( $block['attrs']['style']['layout']['columnStart'] ) && isset( $block['attrs']['style']['layout']['columnSpan'] ) ) {
```

Comment:
We could reduce the uses of `isset` (and make these lines a bit easier on the eye 😅 ) if we assign these to variables outside of the conditions, e.g.:

`$column_start  = isset( $block['attrs']['style']['layout']['columnStart'] ) ? $block['attrs']['style']['layout']['columnStart']  : null`

Reply:
But if they're not set then won't that result in a PHP `undefined index` notice?

Reply:
Oh you covered that with the ternary. Cool 👍 I'll do that